### PR TITLE
Debezium: add PulsarDatabaseHistory for debezium

### PR DESCRIPTION
**Pull request by jiazhai**
### Motivation

add PulsarDatabaseHistory for debezium

### Modifications

add PulsarDatabaseHistory for debezium and test for it.

### Result

ut pass

### CODE REVIEW

File: pulsar-io/debezium/src/main/java/org/apache/pulsar/io/debezium/PulsarDatabaseHistory.java [hunk 1]
```diff
@@ -0,0 +1,243 @@
+/*
+ * Copyright Debezium Authors.
```
Comment (sijie) [inline pulsar-io/debezium/src/main/java/org/apache/pulsar/io/debezium/PulsarDatabaseHistory.java]: this license header might need to be updated.

File: pulsar-io/debezium/src/test/java/org/apache/pulsar/io/debezium/PulsarDatabaseHistoryTest.java [hunk 2]
```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright Debezium Authors.
```
Comment (sijie) [inline pulsar-io/debezium/src/test/java/org/apache/pulsar/io/debezium/PulsarDatabaseHistoryTest.java]: the license header need to be fixed.

**Comment by sijie** [comment]
retest this please

**Comment by sijie** [comment]
run java8 tests

**Comment by sijie** [comment]
the java8 tests seems be failing on the issue I am fixing at #2585

**Comment by sijie** [comment]
run java8 tests

**Comment by sijie** [comment]
#2585 is merged. so rebased this PR to include #2585

**Comment by sijie** [comment]
retest this please

**Comment by sijie** [comment]
retest this please

**Comment by jiazhai** [comment]
rerun java8 tests

**Comment by jiazhai** [comment]
run java8 tests

**Comment by jiazhai** [comment]
run java8 tests

**Comment by jiazhai** [comment]
retest this please

**Comment by jiazhai** [comment]
Seems could no trigger jenkins by add comment like "retest this please, rerun java8 tests"..

**Comment by sijie** [comment]
run java8 tests

**Comment by jiazhai** [comment]
rerun java8 tests

**Pull request merged**